hieradata/common.yaml

```diff
@@ -1311,6 +1311,8 @@ hosts::production::licensify::hosts:
     ip: '10.5.0.18'
 
 hosts::production::management::hosts:
+  db-admin-1:
```

Comment:
Probably. I only added this to get tests to pass, and actually I think I'd rather add an exception in the tests.